flang/test/Lower/OpenMP/distribute.f90

```diff
@@ -0,0 +1,114 @@
+! REQUIRES: openmp_runtime
```

Comment:
Why does this require the runtime?

Reply:
One of the functions contains an ALLOCATE clause that uses `omp_high_bw_mem_alloc` from `omp_lib`, which I think needs the openmp_runtime to work if I'm not wrong.